micromamba/tests/test_info.py

```diff
@@ -71,10 +71,59 @@ def test_not_env(tmp_home, tmp_root_prefix, prefix_selection, existing_prefix):
         else:
             expected_name = name + " (not found)"
         location = prefix
-    print(infos)
 
     assert f"envs directories : {tmp_root_prefix / 'envs'}" in infos
     assert f"environment : {expected_name}" in infos
     assert f"env location : {location}" in infos
     assert f"user config files : {tmp_home / '.mambarc'}" in infos
     assert f"base environment : {tmp_root_prefix}" in infos
+
+
+@pytest.mark.parametrize("base_flag", ["", "--base"])
+@pytest.mark.parametrize("json_flag", ["", "--json"])
+@pytest.mark.parametrize("prefix_selection", [None, "prefix", "name"])
+def test_base_subcommand(
+    tmp_home, tmp_root_prefix, prefix_selection, base_flag, json_flag, monkeypatch
+):
+    monkeypatch.setenv("CONDA_PREFIX", str(tmp_root_prefix))
+
+    if prefix_selection == "prefix":
+        infos = helpers.info("-p", tmp_root_prefix, base_flag, json_flag)
+    elif prefix_selection == "name":
+        infos = helpers.info("-n", "base", base_flag, json_flag)
+    else:
+        infos = helpers.info(base_flag, json_flag)
+
+    items = [
+        "libmamba version",
+        "mamba version",
+        "curl version",
+        "libarchive version",
+        "envs directories",
+        "package cache",
+        "environment",
+        "env location",
+        "user config files",
+        "populated config files",
+        "user config files",
+        "virtual packages",
+        "channels",
+        "platform",
+    ]
+    if base_flag == "--base":
+        if json_flag == "--json":
+            assert all(i not in infos.keys() for i in items)
+            base_environment_path = infos["base environment"].strip()
+        else:
+            assert all(
+                (f"{i} :" not in infos) | (f"\n{i} :" not in infos) for i in items
+            )  # f"\n{i} :" is to handle the case of the "environment" item
+            base_environment_path = infos.replace("base environment :", "").strip()
+        assert os.path.exists(base_environment_path)
+        assert base_environment_path == str(tmp_root_prefix)
+    else:
+        items += ["base environment"]
+        if json_flag == "--json":
+            assert all(i in infos.keys() for i in items)
```

Comment:
Nitpick: This tests that `items` is a subset of `infos.keys()` but  I guess this is sufficient 

This comment also applies elsewhere.